gce_image_import_export_tests/test_suites/import/image_import_test_suite.go

```diff
@@ -164,14 +165,16 @@ func runImageImportWithRichParamsTest(ctx context.Context, testCase *junitxml.Te
 
 	suffix := path.RandString(5)
 	imageName := "e2e-test-image-import-rich-param-" + suffix
+	region, _ := paramhelper.GetRegion(testProjectConfig.TestZone)
 
 	argsMap := map[testsuiteutils.TestType][]string{
 		testsuiteutils.Wrapper: {"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID),
 			fmt.Sprintf("-image_name=%s", imageName), "-data_disk",
 			fmt.Sprintf("-source_file=gs://%v-test-image/image-file-10g-vmdk", testProjectConfig.TestProjectID),
 			"-no_guest_environment", fmt.Sprintf("-family=%v", family), fmt.Sprintf("-description=%v", description),
-			fmt.Sprintf("-network=%v-vpc-1", testProjectConfig.TestProjectID),
-			fmt.Sprintf("-subnet=%v-subnet-1", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-network=global/networks/%v-vpc-1", testProjectConfig.TestProjectID),
```

Comment:
We should still test network/subnet name-only scenarios instead of replacing them with full path scenarios. Both are needed

Reply:
A name-only subnetwork param is covered by another test "runImageImportWithSubnetWithoutNetworkSpecified".
I was trying to cover these scenarios by as less e2e tests as possible because e2e is expensive. I won't test all combination of them.

Please let me know if you don't agree.

Reply:
What about name only network?

Reply:
Let me add one to cover that then.